src/reactive-membrane.ts

```diff
@@ -68,6 +68,24 @@ const defaultValueMutated: ReactiveMembraneMutationCallback = (obj: any, key: Pr
 };
 const defaultValueDistortion: ReactiveMembraneDistortionCallback = (value: any) => value;
 
+export function wrapDescriptor(membrane: ReactiveMembrane, descriptor: PropertyDescriptor, getValue: (membrane: ReactiveMembrane, originalValue: any) => any): PropertyDescriptor {
+    const { set, get } = descriptor;
+    if ('value' in descriptor) {
```

Comment:
Can we use `descriptor.hasOwnProperty('value')` otherwise, can you add a component explaining why?